docs/contributing.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+(For now, just some notes to ick authors)
+
+## Logging
+
+We use vmodule, which provides extra logging levels.  Use `VLOG_1` to mean "info,
```

Comment:
May as well link vmodule to its repo

Reply:
Looks like glog also has some [public docs](https://google.github.io/glog/0.7.1/logging/), but they're not that helpful for understanding the new terms.